mplc/multi_partner_learning.py

```diff
@@ -210,10 +224,8 @@ def fit(self):
         self.learning_computation_time = end - start
         logger.info(f"Training and evaluation on multiple partners: "
                     f"done. ({np.round(self.learning_computation_time, 3)} seconds)")
-        if self.is_save_data:
-            self.history.save_data()
-            # Save the model's weights
-            self.save_final_model()
+        if self.save_folder is not None:
+            self.save_data()  # Save the model weights and the history data
```

Comment:
I think you can do `if self.save_folder` and omit `is not None`, no?

Reply:
I doesn't really like this option. Indeed python can cast None to False, but it also casts 0, or [], or '' to False. It will work